tools/build_defs/pkg/archive.py

```diff
@@ -47,7 +47,7 @@ def __init__(self,
     self.preserve_mtime = preserve_tar_mtimes
 
     self.fileobj = None
-    self.tar = tarfile.open(name=name, mode=mode, fileobj=self.fileobj)
+    self.tar = tarfile.open(name=name, mode=mode, fileobj=self.fileobj, format=tarfile.GNU_FORMAT)
```

Comment:
Rather than hacking archive.py, can pkg.archive.tar be monkey patched at runtime from within our code?

Reply:
Oh, I see, it's a context manager object, so likely no.   Huh, ok, let's discuss more.

Reply:
If you hope to push this upstream you shouldn't set format=tarfile.GNU_FORMAT in the tarfile.open method invocation, but rather define an additional named argument  to the TarFileWriter constructor called "format" with a default value (e.g format=tarfile.DEFAULT), then pass that into the tarfile.open() function.